src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -126,7 +126,7 @@ private void addCommentToGroup(Node n) {
     /** Links a comment to the corresponding node */
     private void linkCommentToNode(Node n) {
       StringBuilder sb = new StringBuilder();
-      String sep = "";
+      String sep = "\n";
```

Comment:
Can you add a test where the Closure code contains:

```
// Hello
// World
/* Hello */
/* World */
```

That is, two subsequent line/C style comments?